src/api/server.js

```diff
@@ -70,19 +70,11 @@ server.submitSuggestQuery = (suggestQuery, callback) => {
 };
 
 server.fetchCsv = (query, callback) => {
-  server.performXhr({
-    url: query.url,
-    data: solrQuery({...query, rows: MAX_INT}, {
-      wt: "csv",
-      "csv.mv.separator": "|",
-      "csv.separator": ";"
-    }),
-    method: "POST",
-    headers: {
-      "Content-type": "application/x-www-form-urlencoded",
-      ...(query.userpass ? {"Authorization": "Basic " + query.userpass} : {}),
-    }
-  }, (err, resp) => {
+
+  const queryString = solrQuery({...query, rows: MAX_INT}, {wt: "csv"});
```

Comment:
Why don't we want the separators specified anymore?

Reply:
The `separator` seems only to change the separators of the result values but the names of the columns still use commas. When I imported the download files with those separators to excel, Excel would not format records reasonably even though we set the different delimiters. In addition, I found solr could return `xlsx` format results. I will explore it and check if it works.